### DISCUSSION TITLE
Too big padding for `:c:expr` inline reference

### DISCUSSION THREAD

**Discussion by marxin** [pradyunsg/furo#486] [Potential Issue] [answered]
I noticed a big left padding for the following snippet:

```rst
.. c:type:: my_type

This works :c:type:`my_type`, but I would like to point
to a pointer of the type (:c:expr:`my_type *`).
```

which renders to:
![Screenshot from 2022-07-25 14-25-03](https://user-images.githubusercontent.com/2658545/180776891-0749155f-b7ef-44bd-98bf-5ba85f07ba83.png)

while other templates render it as:
### alabaster
![Screenshot from 2022-07-25 14-25-46](https://user-images.githubusercontent.com/2658545/180776992-1926983b-51a6-4d62-80fb-ecbf4fbc2c0a.png)

### RTD theme
![Screenshot from 2022-07-25 14-27-48](https://user-images.githubusercontent.com/2658545/180777316-0e21b04c-8f24-44fb-8bef-d7a09affad91.png)

**Comment by BurningEnlightenment** [comment, marked as answer]
This is a duplicate of https://github.com/pradyunsg/furo/discussions/427 (from a technical perspective).
